.gitignore

```diff
@@ -17,3 +17,7 @@ coverage
 .terraform
 *.tfstate
 .terragrunt-cache
+
+# Webpack Output
+public/scripts/index.js
```

Comment:
can you output these into a separate folder under `public/` and then exclude that whole folder?

I'm 👎 about a structure where we're mixing written and generated code in the same place

Reply:
Yep, that makes sense and I agree.  I renamed it to bundle/bundle.js since that's more representative of what it really is since Webpack bundles the CSS and JavaScript into a single bundle.

Reply:
Created folder for webpack build artifacts `public/bundle/`